index.html

```diff
@@ -0,0 +1,130 @@
+        </section>
+
+        <section class="product-category">
+            <h2 class="category-heading text-center">Categories To Bag</h2>
+            <div class="category-display">
+                <div class="card ecommerce-card">
+                    <div class="card-with-text-overlay">
+                        <figure class="card-header">
+                            <img src="/assets/home-images/top.jpg" alt="">
+                        </figure>
+                    </div>
+                    <div class="text-overlay lg-text">Top</div>
+                </div>
+                <div class="card ecommerce-card">
+                    <div class="card-with-text-overlay">
+                        <figure class="card-header">
+                            <img src="/assets/home-images/jeans.jpg" class="img-responsive" alt="">
+                        </figure>
+                    </div>
+                    <div class="text-overlay lg-text">Jeans</div>
+                </div>
+                <div class="card ecommerce-card">
+                    <div class="card-with-text-overlay">
+                        <figure class="card-header">
+                            <img src="/assets/home-images/dresses.jpg" class="img-responsive" alt="">
+                        </figure>
+                    </div>
+                    <div class="text-overlay lg-text">Dresses</div>
+                </div>
+                <div class="card ecommerce-card">
+                    <div class="card-with-text-overlay">
+                        <figure class="card-header">
+                            <img src="/assets/home-images/saree.jpg" class="img-responsive" alt="">
+                        </figure>
+                    </div>
+                    <div class="text-overlay lg-text">Saree</div>
+                </div>
+                <div class="card ecommerce-card">
+                    <div class="card-with-text-overlay">
+                        <figure class="card-header">
+                            <img src="/assets/home-images/kurta.jpg" class="img-responsive" alt="">
+                        </figure>
+                    </div>
+                    <div class="text-overlay lg-text">Kurta Set</div>
+                </div>
+
+            </div>
+        </section>
+    </main>
+
+    <footer>
+        <div class="footer-content ">
+            <h4 class="lg-text text-center">Made With ❤️ By Priyanka Prajapati</h4>
+            <p class="text-center md-text">You can connect with me on:</p>
+            <div class="footer-links header-links x-lg-text">
+                <ul>
+                    <li><a href="https://github.com/prajapati-priyanka"><i class="fa fa-github-square"></i></a></li>
+                    <li> <a href="https://www.linkedin.com/in/priyanka-prajapati-853098146/"><i
+                                class="fa fa-linkedin-square"></i></a></li>
+                </ul>
+            </div>
+        </div>
+    </footer>
+</body>
+</html>
```

Comment:
Just a suggestion ,add new line at the end of the file.

Reply:
okay, I will do that. Thanks.